### PR TITLE
gather bin information from package.json

### LINKED ISSUE #61: Cli tools are not exposed in .bin folder
Due to lots of changes in #53 files from `bin` now is not linked.

The issue is because [here](https://github.com/alexanderGugel/ied/blob/master/lib%2Finstall.js#L123) we now returning not the `package.json` content.

I'm trying to fix it at the next weekend.

### PR DESCRIPTION
**Pull request by just-boris**
fixes #61

### CODE REVIEW

File: test/integration/install.js [hunk 1]
```diff
@@ -35,6 +36,11 @@ describe('install', function () {
       install(__dirname, {_: ['', name + '@' + version]}, function (err) {
         assert.ifError(err)
         require(name)
+        assert(pathIsInside(require.resolve(name), __dirname), 'should resolve package from the target directory')
```
Comment (just-boris) [inline test/integration/install.js]: Added here an extra check that we have got `test/integration/node_modules/mocha`, but not `node_modules/mocha` from dependencies of `ied` itself.

**Comment by alexanderGugel** [comment]
Boom! Another great contribution from @just-boris !

File: lib/expose.js [hunk 2]
```diff
@@ -21,10 +21,11 @@ function symlinkEntry (dir, pkg, cb) {
 
 // Symlinks the bin scripts as specified in the passed in package.json file.
 function symlinkScripts (dir, pkg, cb) {
-  var bin = pkg.bin
+  var info = require(path.join(dir, pkg.uid, 'package.json'))
+  var bin = info.bin
   if (typeof bin === 'string') {
     bin = {}
-    bin[pkg.name] = pkg.bin
+    bin[info.name] = info.bin
```
Comment (rstacruz) [inline lib/expose.js]: shouldnt this be `bin[pkg.name] = info.bin`?

Comment (rstacruz) [inline lib/expose.js]: oops, nevermind, info.name is indeed correct.

Comment (alexanderGugel) [inline lib/expose.js]: :+1:

**Pull request merged**